common/djangoapps/enrollment/tests/test_views.py

```diff
@@ -382,6 +391,10 @@ def test_get_course_details_with_credit_course(self):
     @ddt.unpack
     def test_get_course_details_course_dates(self, start_datetime, end_datetime, expected_start, expected_end):
         course = CourseFactory.create(start=start_datetime, end=end_datetime)
+        # Load a CourseOverview. This initial load should result in a cache
+        # miss; the modulestore is queried and course metadata is cached.
+        __ = CourseOverview.get_from_id(course.id)
+
```

Comment:
@mekkz because I want these tests to verify that with the cache warm, no additional modulestore queries are made.